.github/workflows/test.yaml

```
@@ -10,7 +10,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-
+      - name: Install Tor
+        run: |
+          sudo apt-get install -y tor
```

No need to `run` with a newline.
```suggestion
        run: sudo apt-get install -y tor
```